godb/meta/main.yml

```
@@ -0,0 +1,17 @@
+---
+
+galaxy_info:
+  author: Aaron Jackson
+  namespace: hic
+  description: |
+    Sets up GoDB running in a Docker container with httpd reverse proxy
+  company: Health Informatics Centre, University of Dundee
+
+  license: license (GPL-2.0-or-later, MIT, etc)
```

```suggestion
  license: MIT
```
Otherwise this will fail ansible-lint